src/Entity/User.php

```diff
@@ -56,7 +58,6 @@ public function __construct()
     private string $firstName;
 
     #[ORM\Column(type: Types::STRING, length: 255, nullable: false)]
-    #[Assert\NotBlank]
```

Comment:
display name is used at various places across templates, could be a problem if it's empty